test/correctness/simd_op_check.cpp

```diff
@@ -8,6 +8,14 @@
 
 #include "test/common/halide_test_dirs.h"
 
+// Really, MSVC? *Really?*
+#ifdef max
+#undef max
+#endif
+#ifdef min
+#undef min
+#endif
+
```

Comment:
It might be better to `#define NOMINMAX` before the `#include <windows.h>` in `test/common/halide_test_dirs.h`. That should avoid the min/max macros entirely.